ssh_key.go

```diff
@@ -85,5 +81,15 @@ func (c *Client) FindSSHKey(search string) (*SSHKey, error) {
 		return nil, fmt.Errorf("unable to find %s, zero matches", search)
 	}
 
-	return &keys.Items[found], nil
+	return &keys[found], nil
+}
+
+// DeleteFirewallRule deletes an firewall
+func (c *Client) DeleteSSHKey(id string) (*SimpleResponse, error) {
```

Comment:
As per the errors above, the comment is incorrect. Did VS Code not highlight that to you?